src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -3,6 +3,11 @@ $(document).ready(function() {
     bindUncommonSettingsEvents();
     updateUncommonSettingsInfo();
     hideUncommonPanels();
+
+    $('.dropdown-menu li a').click(function() {
+        $('#questiontype').val(this.id);
+        showNewQuestionFrame(this.id);
```

Comment:
I don't really like coupling the type of the question with its html id, we can explore setting a data attribute on the dropdown option instead?